Samples/iOS-Swift/iOS13-Swift/SwiftUIView.swift

```diff
@@ -1,7 +1,7 @@
 import SentrySwiftUI
 import SwiftUI
 
-struct SwiftUI: View {
+struct SwiftUIView: View {
```

Comment:
`l`: What was the reason you added this change in this PR? Couldn't this be in another PR?

Reply:
It could be, but `iOS13-swift` sample was not compiling, and I want to use it to test arm64e instead of `the iOS-Swift` sample that we use for everything.